### PR TITLE
Reject rounds=0 for SHA1 hashes

### LINKED ISSUE #1617: firebase auth:import do not use the right password
`firebase auth:import` appear to be broken?

With MD5:

```
firebase auth:import users.json --hash-algo=MD5 --rounds=0
```
```
{  
   "users":[  
      {  
         "localId":"uid1",
         "email":"fedtest@gmail.com",
         "emailVerified":true,
         "displayName":"Test Foo Bar",
         "passwordHash":"NzJkNGE1YTViNjA1YzUzYWUxYWM0NzhkOWE0OTc3ZDc=",
         "salt":null
      }
   ]
}
```

This works fine.

But, with SHA1, SHA256, etc.:

```
firebase auth:import users.json --hash-algo=SHA1 --rounds=0
```
```
{  
   "users":[  
      {  
         "localId":"uid1",
         "email":"fedtest@gmail.com",
         "emailVerified":true,
         "displayName":"Test Foo Bar",
         "passwordHash":"NDM3MDkxZjMxYmY1OTgwOWJkMzRjZDBjYzVlNGM3ZGE2ODIwZmU3Nw==",
         "salt":null
      }
   ]
}
```

this don't allow me to login with the password that I choosed. The account is created, but I cannot login. On login I get error is:

```
Error: The password is invalid or the user does not have a password.
```

The plain password for those tests of mine is: `tester5`. I tried with both the latest version of firebase-tools ( 7.2.4 ), and the previous major version as well - same behaviour. 
Am I missing something?

### PR DESCRIPTION
**Pull request by samtstern**
### Description

Fixes #1617
	 
### Scenarios Tested

```
$ npx firebase auth:import foo.json --hash-algo=SHA1 --rounds=0

Error: Must provide valid rounds(1..8192) for hash algori
```

### Sample Commands

N/A

### CODE REVIEW

**Comment by samtstern** [comment]
@bkendall I want someone from the Auth team to comment before we merge to make sure I understood this correctly.

**Comment by bkendall** [comment]
That's super reasonable. Looks good to me otherwise!

**Comment by Federkun** [comment]
But rounds=0 is a valid values for MD5, isn't?

**Comment by samtstern** [comment]
@Federkun yes it's valid for MD5

**Comment by Federkun** [comment]
Won't this PR reject MD5 with rounds zero as well?

**Comment by samtstern** [comment]
Ok we got this confirmation from the auth team:
```
Current Auth backend enforces the round parameter to be [0, 8192] for all MD5/Sha-1/sha-256/sha-512, while round = 0 only works for MD5.
```

File: src/accountImporter.js [hunk 1]
```diff
@@ -175,14 +175,15 @@ var _validateRequiredParameters = function(options) {
         );
       }
       return { hashAlgo: hashAlgo, hashKey: options.hashKey, valid: true };
-    case "MD5":
     case "SHA1":
     case "SHA256":
     case "SHA512":
+      // MD5 is [0,8192] but SHA1, SHA256, and SHA512 are [1,8192]
```
Comment (bojeil-google) [inline src/accountImporter.js]: You removed MD5 from the cases above. I don't think MD5 logic will run here.

Comment (samtstern) [reply src/accountImporter.js]: Phew that was close.

**Pull request merged**